source/tests/iozeug-test/SystemInfo_test.cpp

```diff
@@ -0,0 +1,50 @@
+#include <gmock/gmock.h>
+
+#include <iozeug/SystemInfo.h>
+
+
+using namespace iozeug;
+
+
+class SystemInfo_test : public testing::Test
+{
+public:
+    SystemInfo_test()
+    {
+    }
+
+protected:
+};
+
+TEST_F(SystemInfo_test, pathSeperator)
```

Comment:
I'm not sure whether these tests add any value to the source code, as they basically copy the implementation of the code under test.